app/src/main/java/com/mashup/ui/setting/sns/SNSListScreen.kt

```diff
@@ -0,0 +1,67 @@
+package com.mashup.ui.setting.sns
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.mashup.R
+import com.mashup.URL
+import com.mashup.core.ui.theme.MashUpTheme
+import com.mashup.ui.model.SNSModel
+import com.mashup.ui.setting.SettingViewModel
+import com.mashup.core.common.R as CR
+
+@Composable
+fun SNSListScreen(
+    viewModel: SettingViewModel = viewModel()
+) {
+    val context = LocalContext.current
+
+    val snsList = listOf(
+        SNSModel(stringResource(R.string.facebook), CR.drawable.ic_facebook, URL.FACEBOOK),
+        SNSModel(stringResource(R.string.instagram), CR.drawable.img_instagram, URL.INSTAGRAM),
+        SNSModel(stringResource(R.string.tistory), CR.drawable.ic_tistory, URL.TISTORY),
+        SNSModel(stringResource(R.string.youtube), CR.drawable.ic_youtube, URL.YOUTUBE),
+        SNSModel(stringResource(R.string.mHome), CR.drawable.ic_mashup, URL.MASHUP_UP_HOME),
+        SNSModel(stringResource(R.string.mRecruit), CR.drawable.ic_mashup_dark, URL.MASHUP_UP_RECRUIT),
+    )
```

Comment:
짐작이지만 remember 키워드로 감싸주지 않으면 recomposition 될 때마다 리스트가 생성될 것 같은데 아래처럼 수정하는 것은 어떤가요? :)
SettingScreen 전체적으로 recomposition이 일어나지 않아서 확인은 못했네요 ㅠㅠ

remember 키워드를 사용하도록 변경한다면 SnsModel의 name을 strignRes로 저장하도록 변경해도 좋을 것 같아요!

```suggestion
     val snsList = remember {
        listOf(
            SNSModel(R.string.facebook, CR.drawable.ic_facebook, URL.FACEBOOK),
            ...,
        )
    }
```

Reply:
오 좋은방법인것 같네요!! 아니면 저 리스트를 쭉쭊 위로 올려서  composable func 밖에서 인스턴스 유지는 안좋은 방법일까용? @jaeryo2357  이건제가 잘몰라서 질문~

Reply:
싫어요~ 👎